code/modules/wiremod/components/bci/thought_listener.dm

```diff
@@ -48,7 +48,7 @@
 
 	var/mob/living/owner = bci.owner
 
-	if(!owner || !istype(owner) || !owner.client)
+	if(!owner || !istype(owner) || !owner.client || (owner.stat == DEAD))
```

Comment:
```suggestion
	if(!owner || !istype(owner) || !owner.client || owner.stat => UNCONSCIOUS)
```

no reason for someone unconscious / in hardcrit to be able to communicate stuff too.

Reply:
did you mean `>=`?

Reply:
Yeah, I suppose unconscious people don't think (or have parseable thoughts)

Reply:
This will make it inconsistent with telepathy and slimelink which both allow unconscious individuals to send their thoughts.